linters/v.lua

```diff
@@ -0,0 +1,68 @@
+-- v plugin for lint+
+
+--- INSTALLATION ---
+-- In order to use this linter, please ensure you have the v binary
+-- in your $PATH. For installation notes please see
+-- https://github.com/vlang/v/blob/master/doc/docs.md#installing-v-from-source
+
+--- CONFIG ---
+
+-- config.lint.v_mode: "check" | "check-syntax"
+--   changes the linting mode. check scans, parses, and checks the files 
+--   without compiling the program (default),
+--   check-syntax only scan and parse the files, but then stops. 
+--   Useful for very quick syntax checks.
+-- config.lint.v_args: table[string]
+--   passes the given arguments to v.
+
+--- IMPLEMENTATION ---
+
+local core = require "core"
+local lintplus = require "plugins.lintplus"
+
+local mode = lintplus.config.v_mode or "check"
+if mode ~= "check" and mode ~= "check-syntax" then
+  core.error("lint+/v: invalid v_mode '%s'. "..
+    "available modes: 'check', 'check-syntax'")
+  return
+end
+
+local command
+if mode == "check" then
+  command = lintplus.command {
+    "v",
+    "-check",
+    "-nocolor",
+    "-shared",
+    "-message-limit", "-1",
+    lintplus.args,
+    lintplus.filename
+  }
+elseif mode == "check-syntax" then
+  command = lintplus.args_command({
+    "v",
+    "-check-syntax",
+    "-nocolor",
+    "-shared",
+    "-message-limit", "-1",
+    lintplus.args,
+    lintplus.filename
+  }, "v_args")
+end
+
+lintplus.add("v") {
+  filename = "%.v$",
+  syntax = {
```

Comment:
Why so many options? lite-xl only provides a single syntax file for V and I think shell too. Also instead of adding the same names with different case, it should be instead handled on the changes done on #26 by doing a `lower` call on the compared syntax name. I already merged #26 without taking this into consideration so a separate PR may be needed to add the `string.lower` conversion.

Reply:
Good catch on the `string.lower`, I can create a separate PR for that.
The reason for the different options is, I do not know if there is a standardize way to name syntax plugins.

Also people call different languages by different names. Example, the LSP plugin call `v` `vlang`, which is only used for online searches.

Maybe it was a bit too overzealous, I will take your advice on this.

Reply:
Looking at the [syntax file](https://github.com/lite-xl/lite-xl-plugins/blob/2.1/plugins/language_v.lua#L5) it is called just V and on the lsp it is also identified as [just v](https://github.com/lite-xl/lite-xl-lsp/blob/master/config.lua#L507), the name field itself is that of the language server. The only difference is the case, so feel the confidence to just leave `V` as the syntax name and maybe implement the string.lower when you get some time.